misc/specs/atrium-specs-common/src/main/kotlin/ch/tutteli/atrium/specs/integration/AnyAssertionsSpec.kt

```diff
@@ -132,6 +148,16 @@ abstract class AnyAssertionsSpec(
                         expectSubject.isNotSameFun(1)
                     }.toThrow<AssertionError> { messageContains(IS_NOT_SAME.getDefault()) }
                 }
+                it("${isNoneOf.name} throws AssertionError") {
+                    expect {
+                        expectSubject.isNoneOfFun(1, emptyArray())
```

Comment:
Please use two numbers here (I know it does not 100% fit the context but I think that's fine -- or create another context if you want)